File: c#/almacen.cs
using System.Collections.Generic;
public class almacen{
	private int _idAlmacen;
	private string _nombre;
	private List<ingrediente> _stock;

	public almacen(){}

	public almacen(int _idAlmacen, string _nombre, List<ingrediente> _stock){
		this._idAlmacen = _idAlmacen;
		this._nombre = _nombre;
		this._stock = _stock;
	}

	public int IdAlmacen{
		get{
			return _idAlmacen;
		}
		set{
			_idAlmacen = value;
		}
	}

	public string Nombre{
		get{
			return _nombre;
		}
		set{
			_nombre = value;
		}
	}

	public List<ingrediente> Stock{
		get{
			return _stock;
		}
		set{
			_stock = value;
		}
	}
}

File: java/Almacen.java
import java.util.ArrayList;
public class Almacen{
	private int idAlmacen;
	private String nombreAlm;
	private ArrayList<Ingrediente>listIngrediente;
	
	public Almacen(){
		listIngrediente=new ArrayList<Ingrediente>();
	}
	public Almacen(int idAlmacen,String nombreAlm){
		this.idAlmacen=idAlmacen;
		this.nombreAlm=nombreAlm;
		this.listIngrediente=new ArrayList<Ingrediente>();
	}
	public void setidAlmacen(int idAlmacen){
		this.idAlmacen=idAlmacen;
	}
	public int getidAlmacen(){
		return this.idAlmacen;
	}
	
	public void setnombreAlm(String nombreAlm){
		this.nombreAlm=nombreAlm;
	}
	public String getnombreAlm(){
		return this.nombreAlm;
	}
	
	public void setlistIngrediente(ArrayList<Ingrediente> listIngrediente){
		this.listIngrediente=listIngrediente;
	}
	public ArrayList<Ingrediente> getlistIngrediente(){
		return this.listIngrediente;
	}
	
}